client/src/pages/Communities/Create/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import {
+  Button,
+  Checkbox,
+  FormControlLabel,
+  FormGroup,
+  TextField,
+  Typography,
+} from '@mui/material'
+
+const Page = styled.main`
+  width: 100%;
+  padding: 0.5rem;
+  margin-top: 5rem;
+`
+
+const Form = styled.form`
+  width: 95%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  border-radius: 8px;
+  background: rgba(33, 33, 33, 0.5);
+  padding: 2rem;
+`
+
+const CreateCommunity = () => {
+  const handleFormSubmit: React.FormEventHandler = (e) => {
+    e.preventDefault()
+    const formData = new FormData(e.target as any)
+
+    // @ts-ignore
+    console.dir([...formData.values()])
```

Comment:
TODO: call the API with these values and create the community. Do this via the Redux Toolkit _stuff_ so that we have the community in the Redux store and can pull from that on the pages throughout the application